keyboards/canary/canary60rgb/v2/keyboard.json

```diff
@@ -0,0 +1,222 @@
+{
+    "keyboard_name": "CANARY60RGB V2",
+    "manufacturer": "CANARY",
```

Comment:
Already specified in `canary60rgb/info.json`, either: 
1. keep these settings in `canary60rgb.info.json`, which would require these settings removed from `v1/keyboard.json` and `v2/keyboard.json`, as well as remove `"maintainer"` from `v1/keyboard.json` 
2. move `"maintainer"` from `canary60rgb/info.json` to `v2/keyboard.json` and delete `canary60rgb.info.json`

Reply:
Can you tell me which option is preferred?

Reply:
Done. Please check again.